src/abstractdataframe/prettytables.jl

```diff
@@ -0,0 +1,139 @@
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# Description
+# ==============================================================================
+#
+#     This file contains the functions that build the interface with
+#     PrettyTables.jl.
+#
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Internal function to print DataFrames using PrettyTables.jl.
+function _pretty_table(io::IO, df::AbstractDataFrame;
+                       allrows::Bool = !get(io, :limit, false),
+                       allcols::Bool = !get(io, :limit, false),
+                       rowlabel::Symbol = :Row,
+                       summary::Bool = true,
+                       eltypes::Bool = true,
+                       rowid=nothing,
+                       truncstring::Int = 32,
+                       kwargs...)
```

Comment:
so `splitcols` is not supported - right? (it does not have to, I just want to know)

Reply:
No, it's not! I really do not know how to support it. PrettyTables currently do not tell you how many columns were actually printed.